lib/ansible/modules/packaging/os/service_facts.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/python
+# (c) 2017, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# originally copied from AWX's scan_services module to bring this functionality
+# into Core
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+import re
+from ansible.module_utils.basic import * # noqa
```

Comment:
Please import just what you need
http://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#python-imports

Reply:
:+1: